static/src/partials/defaultStyle.html

```diff
@@ -34,6 +34,7 @@
     height: 100%;
     width: 100%;
     overflow: hidden;
+    background-color: white;
```

Comment:
I'm updating this page so that the transition page is the same color as the backgrounds for the user pages content-info etc.